base/float.jl

```diff
@@ -742,6 +742,28 @@ significand_mask(::Type{Float16}) = 0x03ff
 @pure exponent_bits{T<:AbstractFloat}(::Type{T}) = sizeof(T)*8 - significand_bits(T) - 1
 @pure exponent_bias{T<:AbstractFloat}(::Type{T}) = Int(exponent_one(T) >> significand_bits(T))
 
+## TwicePrecision utilities
+for (F, T, n) in ((Float16, UInt16, 5), (Float32, UInt32, 12), (Float64, UInt64, 26))
```

Comment:
the significance of these `n` values is half the number of mantissa bits, right?